boards/BMS/Src/App/App_Accumulator.c

```diff
@@ -194,3 +200,40 @@ void App_Accumulator_RunOnTick100Hz(struct Accumulator *const accumulator)
             break;
     }
 }
+
+bool App_Accumulator_CheckFaults(
+    struct BmsCanTxInterface *const can_tx,
+    struct Accumulator *const       accumulator,
+    struct TractiveSystem *const    ts)
+{
+    uint8_t throwaway_segment = 0U;
+    uint8_t throwaway_loc     = 0U;
+
+    float max_allowable_cell_temp = MAX_CELL_DISCHARGE_TEMP_DEGC;
+    float min_allowable_cell_temp = MIN_CELL_DISCHARGE_TEMP_DEGC;
+
+    // if we are charging, max cell temp is 45C not 60C
+    if (App_TractiveSystem_GetCurrent(ts) < 0.0f)
```

Comment:
```suggestion
    if (App_TractiveSystem_GetCurrent(ts) < -3.0f)
```
Can you add a little margin for error here. 3A should be enough to account for any error and is still much lower than our max charge current